packages/common/src/apiutil/ServerConfig.ts

```diff
@@ -2,18 +2,29 @@ import fs from "node:fs"
 import { LevelWithSilent } from "pino"
 import { expressLogger, verbosity } from "./logging.js"
 
+// Define the possible values of database creation both as a type, and as an array of strings
+const DatabaseCreationValues = ["always", "never", "if-not-exists"] as const
```

Comment:
Don't we use the same constants also for the creation of repositories? If so, should we use a more generic name?